src/handlebars/faq.handlebars

```diff
@@ -0,0 +1,188 @@
+{{> header title='FAQ | ' }}
+
+<!-- TODO Style should match installation page, it doesn't quite... -->
```

Comment:
Probably worth running the output through Grammarly or similar, there's a few missing punctuation marks that could possibly go in on this doc

Reply:
I don't want to install grammerly on my machine, but I'll see if someone else has it. Otherwise, you're welcome to point things out and/or suggest updates specifics ;-)

Reply:
Have resolved most of the concerns that showed up